openlibrary/core/models.py

```diff
@@ -428,7 +428,7 @@ def from_isbn(cls, isbn):
             return None  # consider raising ValueError
 
         isbn13 = to_isbn_13(isbn)
-        isbn10 = isbn_13_to_isbn_10(isbn)
+        isbn10 = isbn_13_to_isbn_10(isbn13)
```

Comment:
the `isbn_13_to_isbn_10` returns None if an ISBN 10 is passed therefore we are losing the ISBN 10